mono/metadata/icall.c

```
@@ -3063,7 +3076,8 @@ ves_icall_InternalInvoke (MonoReflectionMethod *method, MonoObject *this_arg, Mo
 		}
 
 		if (this_arg) {
-			if (!mono_object_isinst (this_arg, m->klass)) {
+			if (!mono_object_isinst_checked (this_arg, m->klass, &error)) {
+				mono_error_cleanup (&error);
```

We shouldn't swallow this error.